app-vim/rust-vim/rust-vim-1_pre20210215.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit vim-plugin
+
+COMMIT="87c745d8d506fc1eecc1d81df15d5bde1658a2fc"
+MY_PN="${PN/-/.}"
+
+DESCRIPTION="Vim configuration for Rust"
+HOMEPAGE="https://www.rust-lang.org https://github.com/rust-lang/rust.vim"
+SRC_URI="https://github.com/rust-lang/rust.vim/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
```

Comment:
The previous `|| ( MIT Apache-2.0 )` seems right to me, same that dev-lang/rust uses -- you'll see this package has both license files.

I know github lists Apache-2.0, but github doesn't really care for nuances.